docker-compose.yml

```diff
@@ -10,40 +10,24 @@ services:
     image: pelias/baseimage
```

Comment:
this file is really hard to review because the diff is all out-of-order, is it possible to restore the order so I can easily see only what changed?